tools/nuttx-gdbinit

```diff
@@ -455,7 +441,26 @@ define info_nxthreads
 end
 
 define nxcontinue
-  printf "nxcontinue\n"
   _restore_current_tcb
   continue
 end
+
+define info threads
+  info_nxthreads
+end
+
+define thread
+  _save_current_tcb
+  if ($argc == 0)
+    _running_task
+    set $pid = $rtcb->pid
+  else
+    set $pid = $arg0
+  end
+
+  nxthread $pid
```

Comment:
let's change nxthread to thread directly

Reply:
Done